src/input.rs

```diff
@@ -75,3 +128,8 @@ pub fn is_mouse_button_released(btn: MouseButton) -> bool {
 
     context.mouse_released.contains(&btn)
 }
+
+/// Convert a position in pixels to a position in the range [-1; 1].
+fn convert_to_local(pixel_pos: Vec2) -> Vec2 {
+    Vec2::new(pixel_pos.x / screen_width(), pixel_pos.y / screen_height()) * 2.0 - Vec2::new(1.0, 1.0)
+}
```

Comment:
You could run `rustfmt` on changed files

Reply:
done